### ISSUE TITLE
If multiple tokens of the same name attack, one becoming unblocked will cause it to be near indistinguishable from the other tokens.Workaround:place your cursor over the blocked token in the game log, and it will highlight that token on the battlefield.

### ISSUE THREAD

**Issue by vorpal-buildbot** [PennyDreadfulMTG/modo-bugs#570]
From Bug Blog.
Bug Blog Text: If multiple tokens of the same name attack, one becoming unblocked will cause it to be near indistinguishable from the other tokens.Workaround:place your cursor over the blocked token in the game log, and it will highlight that token on the battlefield.
Affects:

**Comment by stash86** [comment]
Duplicate of #396
